show/main.py

```diff
@@ -464,6 +464,72 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+@cli.group('storm-control')
+def storm_control():
+    """ show storm-control """
+    pass
+@storm_control.command('all')
+def storm_control_all():
+    """ Show storm-control """
+
+    header = ['Interface Name', 'Storm Type', 'Rate (kbps)']
+    body = []
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+
+    table = config_db.get_table('PORT_STORM_CONTROL')
+
+    #To avoid further looping below
+    if not table:
+        return
```

Comment:
include a msg before returning

Reply:
Returning the prompt when there is no content to display.